__tests__/app.test.js

```diff
@@ -1,14 +1,55 @@
 const pool = require('../lib/utils/pool');
 const setup = require('../data/setup');
-// const request = require('supertest');
-// const app = require('../lib/app');
+const request = require('supertest');
+const app = require('../lib/app');
+
+const { blocks } = require('../lib/blocks-data');
+const { potions } = require('../lib/potions-data');
 
 describe('backend-express-template routes', () => {
   beforeEach(() => {
     return setup(pool);
   });
-  it('example test - delete me!', () => {
-    expect(1).toEqual(1);
+
+  it('/blocks should return a list of blocks', async () => {
+    const res = await request(app).get('/blocks');
+    const expected = blocks.map((block) => {
+      return { id: block.id, name: block.name };
+    });
+    expect(res.body).toEqual(expected);
+  });
+
+  it('/blocks/:id should return the block detail', async () => {
+    const res = await request(app).get('/blocks/1');
+    const stone = {
+      id: '1',
+      itemid: 'minecraft:stone',
+      name: 'Stone',
+      stackable: true,
+      size: 64,
+      url: 'https://minecraftitemids.com/item/128/stone.png',
+    };
+    expect(res.body).toEqual(stone);
+  });
+
+  it('/potions should return a list of potions', async () => {
+    const res = await request(app).get('/potions');
+    const expected = potions.map((potion) => {
```

Comment:
nice job implementing `.map` instead of a for loop. Remember, since you know you want only id and name you can destructure them in your map ex. `potions.map(({ id, name })=> ({ id: id, name: name }))` .  Julie just went over this and it's totally fine to do it the way you have it, just wanted to put some ideas in your head for the future! Great work!